wrappers/java/src/main/java/org/ensembl/hive/RunWrapper.java

```diff
@@ -48,24 +48,26 @@ public static void main(String[] args) throws Exception {
 			System.err.println("The debug level '" + args[3] + "' is not an integer. Aborting");
 			System.exit(1);
 		}
+        // NOTE: curret version of library suppose only one debug level fod SimpleLogger,
```

Comment:
This block you've commented out was actually useful: depending on the selected log-level, more or less debug information would be printed, possibly some very detailed stuff ("TRACE")

Reply:
This library contains several classes, so we need just to use another class, not SimpleLogger, I suppose to do it in another pull request.
Here is description that Simple logger doesn't support levels https://www.slf4j.org/manual.html
, with the pic of different classes of the lib

Reply:
I don't follow. In that page you can see they call `logger.debug`, `logger.info`. And then on https://www.slf4j.org/faq.html there is `logger.error` too